packages/edit-site/src/components/sync-state-with-url/use-init-edited-entity-from-url.js

```diff
@@ -11,16 +11,17 @@ import { privateApis as routerPrivateApis } from '@wordpress/router';
  */
 import { store as editSiteStore } from '../../store';
 import { unlock } from '../../lock-unlock';
-import {
-	TEMPLATE_POST_TYPE,
-	TEMPLATE_PART_POST_TYPE,
-	NAVIGATION_POST_TYPE,
-	PATTERN_TYPES,
-} from '../../utils/constants';
 
 const { useLocation } = unlock( routerPrivateApis );
 
-export function useInitEditedEntity( { postId, postType } ) {
+const postTypesWithoutParentTemplate = [
+	'wp_template',
+	'wp_template_part',
+	'wp_block',
+	'wp_navigation',
+];
```

Comment:
Constants are also available for these. As mentioned, not a huge deal. It's always something we can follow up on anyway.

```js
import {
	TEMPLATE_POST_TYPE,
	TEMPLATE_PART_POST_TYPE,
	NAVIGATION_POST_TYPE,
	PATTERN_TYPES,
} from '../../utils/constants';
```

```js
const postTypesWithoutParentTemplate = [
	TEMPLATE_POST_TYPE,
	TEMPLATE_PART_POST_TYPE,
	PATTERN_TYPES.user,
	NAVIGATION_POST_TYPE,
];
```

Reply:
I won't touch it here but the `PATTERN_TYPES` const is very confusing. I'd expect a `PATTERN_POST_TYPE` const like the others instead, we have a constant that has two keys that refer to very different things.